### DISCUSSION TITLE
TrackingNet

### DISCUSSION THREAD

**Discussion by zapatistas** [STVIR/pysot#449] [Code] [answered]
Hello,

  I was wondering if and how it is possible to train the pre-trained models on the TrackingNet data set. Is it possible to just use one of the pre-processing methods specified in the training_dataset folder and then train/test the network or is there any other way to do that?

Thank you.

**Comment by ZhiyuanChen** [comment, marked as answer]
The pre-trained model was trained on datasets including ybb, why do you want to retrain it with trackingnet?

**Reply by zapatistas** [reply]
As a part of an educational project for object tracking, I wanted to try and do transfer learning using a pre-trained SiamRPN++ on the trackingnet data set, explore the effects of that data set in the performance of the model and also try to fine-tune it etc, would you recommend another data set for that?

**Reply by ZhiyuanChen** [reply]
> As a part of an educational project for object tracking, I wanted to try and do transfer learning using a pre-trained SiamRPN++ on the trackingnet data set, explore the effects of that data set in the performance of the model and also try to fine-tune it etc, would you recommend another data set for that?

Given your situation, I would argue that any dataset that is NOT used in pretraining should be fine.

**Reply by zapatistas** [reply]
In the case that I want to use trackingnet, since this is the data set we decided to work on with in my group, is it possible to use some functions of pysot, for example for pre-processing and creating .json files etc  or do I have to implement mine and also if i get the pre-processing right could I then use the already implemented .py files for training/testing? As stated in another discussion for using custom data sets, I tried using some of the .py files, for pre-processing, meant for other data sets, but I could not make them work on trackingnet.

**Reply by ZhiyuanChen** [reply]
It is your freedom to use any code, or not. TrackingNet is a subset of YBB, which has been used for training the pretrained module.

**Reply by zapatistas** [reply]
I was not aware of that, thank you.

**Reply by ZhiyuanChen** [reply]
It's ok. Though I would encourage you to study your dataset before you make the decision of using it
